dbos/_dbos.py

```diff
@@ -409,6 +409,21 @@ def _launch(self) -> None:
             dbos_logger.error(f"DBOS failed to launch: {traceback.format_exc()}")
             raise
 
+    @classmethod
+    def reset_system_database(cls) -> None:
+        """
+        Reset the DBOS system database. Useful for resetting the state of DBOS between tests.
```

Comment:
I think this description can be clear about we're terminating all connections to the system DB and dropping the system database.